compose/control-plane/config.json

```diff
@@ -18,6 +21,96 @@
         "metric_system_name": "hits",
         "delta": 1
       }
-    ]
+    ],
+    "auth_config": {
+      "path": "static/threescale_wasm_auth.wasm",
+      "wasm_config": {
+        "system": {
+          "cluster_name": "a_cluster",
+          "url": "https://a-system-url/",
+          "token": "a system token",
+          "timeout": 5
+        },
+        "backend": {
+          "cluster_name": "3scale-saas-backend",
+          "url": "https://su1.3scale.net/",
+          "timeout": 5
+        },
+        "services": [
```

Comment:
There are two services registered in the same service?

Reply:
keep in mind that this is the configuration for the wasm module, not the cp, so this is the listener configuration for the module - it just declares that for this listener the module can discriminate between two different services, but I just left that because that is the example configuration from the wasm module, and we can just limit this to the `web.app` service in this case